packages/react-components/react-message-bar/library/src/components/MessageBar/useMessageBarReflow.ts

```diff
@@ -6,9 +6,6 @@ export function useMessageBarReflow(enabled: boolean = false) {
   const { targetDocument } = useFluent();
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33409&commitId=cb341c5e8f19306aa8787b984988a6af92f27ea9&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.chromium.png|3| Changed|
|Avatar Converged.Badge Mask RTL.chromium.png|1| Changed|

</details>


  <details><summary><b>Drawer</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay High Contrast.chromium.png|4916| Changed|
|Drawer.Full Overlay Dark Mode.chromium.png|991| Changed|
|Drawer.overlay drawer full.chromium.png|3302| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>